code/upaya/app/assets/javascripts/homepage.js

```diff
@@ -0,0 +1,13 @@
+$(document).on('page:change', function () {
+  var maxH = 0;
+  $('.same-height-panel').each( function () {
+    var thisH = $(this).height();
+    if (thisH > maxH) {
+      maxH = thisH;
+    }
+  });
+
+  $('.same-height-panel').each( function () {
+    $(this).css('minHeight',maxH+'px');
+  });
+});
```

Comment:
I'm not a frontend expert, but it seems like you can do this without using Javascript, using Flexbox, or tables if IE 8 and 9 support is needed: http://stackoverflow.com/questions/2997767/how-do-i-keep-two-divs-that-are-side-by-side-the-same-height